README.md

```diff
@@ -25,6 +25,18 @@
 * [ml5 -> Unet](https://ml5js.org/reference/api-unet/)
 * [p5js](https://p5js.org/)
 
+<div align="center">
+
+[![forthebadge](https://forthebadge.com/images/badges/open-source.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com) 
+[![forthebadge](https://forthebadge.com/images/badges/makes-people-smile.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/powered-by-coffee.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/winter-is-coming.svg)](https://forthebadge.com)
+
+</div>
+
+## Project Demo-
+![Demo1](assets/demo1.png)
```

Comment:
For now, the demo screenshot is not required. Once this project gets a good amount of UI revamp, then feel free to raise a PR & do that.

So, remove the screenshots.